.github/workflows/build_python.yml

```diff
@@ -34,9 +34,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - pyversion: ${{ github.event.schedule == '0 15 * * *' && "pypy3" }}
-          - pyversion: ${{ github.event.schedule == '0 17 * * *' && "python3.10" }}
-          - pyversion: ${{ github.event.schedule == '0 19 * * *' && "python3.12" }}
+          - pyversion: ${{ github.event.schedule == '0 15 * * *' && 'pypy3' }}
+          - pyversion: ${{ github.event.schedule == '0 17 * * *' && 'python3.10' }}
+          - pyversion: ${{ github.event.schedule == '0 19 * * *' && 'python3.12' }}
```

Comment:
Single quotation mark will work like the following.

https://github.com/apache/spark/blob/84c5b919d99872858d2f98db21fd3482f27dcbfc/.github/workflows/build_and_test.yml#L202